plugin/evm/vm_upgrade_bytes_test.go

```diff
@@ -249,7 +256,7 @@ func TestVMStateUpgrade(t *testing.T) {
 		Code:          upgradedCode,
 	}
 
-	upgradeTimestamp := time.Unix(10, 0) // arbitrary timestamp to perform the network upgrade
+	upgradeTimestamp := time.Now() // arbitrary timestamp to perform the network upgrade
```

Comment:
nit: should we specify the required upgrade (`DefaultGenesisTime`) + some arbitrary amount instead of time.Now?

Reply:
yup changed it